javascript/operators/pipeline.json

```diff
@@ -1,59 +0,0 @@
-{
-  "javascript": {
-    "operators": {
-      "pipeline": {
-        "__compat": {
-          "description": "Pipeline operator (<code>|></code>)",
-          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/Pipeline_operator",
```

Comment:
This one is actually implemented https://bugzilla.mozilla.org/show_bug.cgi?id=1405943 but it's behind build flag and can't be enabled in release builds. I think it's safe to remove this.

Reply:
Since this PR was opened, the pipeline operator implementation has been removed from Firefox: https://bugzilla.mozilla.org/show_bug.cgi?id=1708546.

@Elchi3 Any sense of what should happen to the content for this one? I know there was a lot of buzz around the pipeline operator at one point, but I don't know if that's enough to keep the content around, if implementation is going backwards.

(The other stuff in this PR doesn't seem to have content, so it should be safe to remove.)

Reply:
hm, I'd say let's remove this for now then.

Reply:
Opened https://github.com/mdn/content/pull/5394

Reply:
https://github.com/mdn/content/pull/5394 has been removed, so it seems like it’s time to go ahead and merge this PR?

Reply:
Yeah, let's merge this 👍